### ISSUE TITLE
Support stats type NanCount

### ISSUE THREAD

**Issue by pford** [CARTAvis/carta-backend#161]
For CyberSKA requirement, no corresponding stat in casacore.

**Comment by veggiesaurus** [comment]
@arusstaylor can you comment on whether this is essential? Haven't seen a NaN count statistic in viewers other than CyberSKA before.

**Comment by arusstaylor** [comment]
I don’t think this is essential as long as the pixel count does not include Nans. 

Russ

**Comment by veggiesaurus** [comment]
@pford I believe the cassacore pixel count excludes NaNs at this point. Should we close this issue then?

**Comment by arusstaylor** [comment]
Fine with me to close.

Russ

**Comment by pford** [comment]
Yes, NumPixels excludes NaNs.  Will close.  This CARTA::StatType should be removed from the ICD.

**Comment by pford** [comment]
Needed for release 1.2 based on testing feedback.

**Comment by pford** [comment]
As I understand it, this is extremely difficult in casacore.  Images are masked so that unmasked pixels may be used for calculations, which means that for a region, a pixel may be masked because it is NaN, it is outside the region, or both.  Therefore impossible to count "in the region but value is NaN".

**Comment by veggiesaurus** [comment]
@pford I guess we can icebox this for now then
